app/src/main/java/org/fossasia/openevent/general/event/EventDao.kt

```diff
@@ -22,7 +22,7 @@ interface EventDao {
     fun getAllEvents(): Flowable<List<Event>>
 
     @Query("SELECT * from Event WHERE id = :id")
-    fun getEvent(id: Long): Flowable<Event>
+    fun getEvent(id: Long): Single<Event>
```

Comment:
Please revert. This breaks live update of the event

Reply:
what do you mean by live update? I haven't understood why would we want a continuous stream of the same data

Reply:
You don't get the same data. You get it if something in DB has changed

Reply:
https://en.wikipedia.org/wiki/Single_source_of_truth

Reply:
https://codelabs.developers.google.com/codelabs/android-persistence/#5